modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ConfigurationModelToDDMFormConverter.java

```diff
@@ -275,7 +282,10 @@ else if (type == ExtendedAttributeDefinition.LOCALIZED_VALUES_MAP) {
 			return DDMFormFieldType.LOCALIZABLE_TEXT;
 		}
 
-		if (!SetUtil.isEmpty(ddmFormFieldOptions.getOptionsValues())) {
```

Comment:
Removing this condition will cause a regression for fields that have their options defined directly in the configuration class. See [CaptchaConfiguration](https://github.com/liferay/liferay-portal/blob/4526402a5c2d5f94d485f3a72878c09b1ccee669/modules/apps/captcha/captcha-api/src/main/java/com/liferay/captcha/configuration/CaptchaConfiguration.java#L60-L70) as an example. I think it would work to just include both checks.